pkg/util/iptables/iptables.go

```diff
@@ -175,6 +175,12 @@ const WaitString = "-w"
 // WaitSecondsValue a constant for specifying the default wait seconds
 const WaitSecondsValue = "5"
 
+// WaitIntervalString a constant for specifying the wait interval flag
+const WaitIntervalString = "-W"
+
+// WaitIntervalUsecondsValue a constant for specifying the default wait interval useconds
+const WaitIntervalUsecondsValue = "10000"
```

Comment:
see the description to understand this value

Reply:
That's 100 Hz.  That feels aggressive - why not 10Hz ?

Reply:
😅 because I was thinking that every message represented an attempt to acquire the lock. Seems that iptables send the message every 10 attempts.
Let me change it to 10Hz